src/mds/MDSRank.cc

```diff
@@ -743,6 +743,7 @@ void MDSRankDispatcher::tick()
 
   // ...
   if (is_clientreplay() || is_active() || is_stopping()) {
+    server->clear_laggy_clients();
```

Comment:
this will reset the set before going to find_idle_sessions and evict_cap_revoke_non_responders at every tick(), not required to be done in both of them separately as its

```  // ...
  if (is_clientreplay() || is_active() || is_stopping()) {
    server->clear_laggy_clients();
    server->find_idle_sessions();
    server->evict_cap_revoke_non_responders();
    locker->tick();
  }```